cli/chaosli/cmd/root.go

```diff
@@ -51,14 +51,8 @@ func Execute() {
 }
 
 func init() {
-	cobra.OnInitialize(initConfig)
-	cobra.OnInitialize(func() {
-		var err error
```

Comment:
this is called only on each subcommand's init func but not when we don't use a command.